portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -80,12 +80,15 @@
 
 <jsp:include page="global/header.jsp" flush="true"/>
 
+<script type="text/javascript" src="js/src/modifyQuery.js?<%=GlobalProperties.getAppVersion()%>"></script>
 
 <script>
 window.appVersion = '<%=GlobalProperties.getAppVersion()%>';
     
 window.historyType = 'memory';
 
+window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
```

Comment:
@inodb Actually this variable is already in `template.tag`.

I had a discussion with @alisman already about this. It is not trivial to include `template.tag` in those files (`visualize.jsp` and `cross_cancer_results.jsp`). So this is a temporary workaround for now.